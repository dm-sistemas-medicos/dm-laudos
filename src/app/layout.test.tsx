import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import RootLayout from "./layout";

describe("Root layout", () => {
  it("renders the layout", () => {
    const { container } = render(<RootLayout>children</RootLayout>);

    expect(container).toMatchSnapshot();
  });
});
