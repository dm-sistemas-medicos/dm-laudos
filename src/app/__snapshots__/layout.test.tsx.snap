// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root layout renders the layout 1`] = `
<div>
  <html
    lang="pt-BR"
  >
    <body
      class="className"
    >
      children
    </body>
  </html>
</div>
`;
